concepts/basics/about.md

```diff
@@ -0,0 +1,108 @@
+# About
+
+[Lua](https://lua.org) is a simple yet powerful, efficient, lightweight, embeddable scripting language. It is designed, implemented, and maintained at [PUC-RIO](https://www.puc-rio.br/) in Brazil since 1993 by the team of Roberto Ierusalimschy, Waldemar Celes and Luiz Henrique de Figueiredo.
+
+## Comments
+
+Singleline comments in Lua start with `--` and multiline comments start with `--[[` and end with `]]`.
+```lua
+-- This is a single line comment.
+
+--[[This is
+a multiline
+comment
+]]
+```
+
+## Variables
+
+There are three types of variables in Lua: global variables, local variables and table fields. To keep things simple only local variables are explained here. By default all variables are global so we have to declare local variables as such with the keyword `local`.
+```lua
+local x
+```
+
+A value can be assigned to a variable by using the `=` operator.
+```lua
+local x = 1
+```
+
+The scope of a `local` variable is limited to the block where it is declared. A block can be the body of a control structure, the body of a function or a file or string where the variable is declared.
+```lua
+local x = 1
+
+if x == 1 then
+    local x = 2
+    print(x) -- prints 2
+end
+
+print(x) -- prints 1
+```
+
+## Functions
+
+A function is declared with the keyword `function` and, like variables, is global by default too. A function definition has a name, a list of parameters and a body. The functions exits when the keyword `return` is reached and returns all values listed after it. Without a specified `return`, the function will return `nil`.
```

Comment:
```suggestion
A function is declared with the keyword `function` and, like variables, is global by default. A function definition has a name, a list of parameters and a body. The functions exits when the keyword `return` is reached and returns all values listed after it. Without a specified `return`, the function will return `nil`.
```

The "too" seems redundant to me because of the "like variables"